spec/models/user_spec.rb

```diff
@@ -22,5 +22,9 @@
 
       it { is_expected.to_not be_valid }
     end
+
+    describe 'database columns' do
```

Comment:
Thanks, I didn't know about it. It will be better if I delete spec for db column.